### PR TITLE
Adapt for new ForcesPro version >=6

### PR DESCRIPTION
**Pull request by t-thanh**
New name function casadi2forces replaced by adtool2forces, generate code with legacy interface, https://forces.embotech.com/Documentation/back_compatibility/index.html#changes-from-version-6-0-0

### CODE REVIEW

**Review by JulioLP — COMMENTED** [review]
Is there a reason for using these placeholders? I prefer to leave the file structure and .gitignore file as previously was. Since the "PerchingSolver" folder is not synced, there is no reason to upload these empty folders to github.

File: solver_generation/generate_solver.m [hunk 1]
```diff
@@ -66,10 +66,13 @@
...
 codeoptions.solver_timeout = 1;
 
+%Add to generate code for ForcesPro version 5
+codeoptions.legacy_interface = 1;
+
 codeoptions.avx = 1;
 codeoptions.sse = 1;
 
-codeoptions.platform = 'Gnu-x86_64';
+%codeoptions.platform = 'Gnu-x86_64'; %comment out if only use for simulation
```
Comment (JulioLP) [inline solver_generation/generate_solver.m:75]: In general people will test this solver in their laptops, and only change the platform when they know what they do. If there is no conflict with the new version I prefer to leave this as before.

**Comment by JulioLP** [comment]
In general LGTM, apart from the 2 comments that I left. Can you confirm that all the example cases work properly and show me  the computational time for them? Thanks!

**Comment by t-thanh** [comment]
@JulioLP : For the placeholder, without it, the folder doesn't exist and in my case, the generate_solver.m will fail at copy file.
For the platform, with the option appear, I have the return of license only support simulation, even I change to the correct platform of the host.

**Comment by JulioLP** [comment]
I checked the PR in my laptop and the results look similar. Thanks for the effort!

**Pull request merged**